auth.go

```
@@ -3,11 +3,39 @@ package main
+// that the required fields are present, and persists it to the config file. It
+// is shared by the interactive TUI auth screen and the `buzz auth login`
+// command so both accept identical input and report identical errors.
+func parseAndSaveCredentials(input string) (*Config, error) {
+	input = strings.TrimSpace(input)
+	if input == "" {
+		return nil, fmt.Errorf("please enter your credentials")
+	}
+
+	var config Config
+	if err := json.Unmarshal([]byte(input), &config); err != nil {
+		return nil, fmt.Errorf("invalid JSON format: %v", err)
```

golangci-lint (2.12.2)

[error] non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)